### PR TITLE
Initial commit for Anroid_page

### PR DESCRIPTION
**Pull request by jdspringr07**
This branch is the initial android place. It's just the scaffolding for developing the android help page once we get someone to provide further information.

Addresses #119

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -2,6 +2,9 @@ class PagesController < ApplicationController
   def about
   end
 
+  def android
+  end
+
```
Comment (jdspringr07) [inline app/controllers/pages_controller.rb]: Not sure why this is complaining. This is following the format of the rest of pages_controller.rb

**Comment by rickr** [comment]
Could you make it so this page is accessible via the url `/getting-started/android`?

I'd also like to get this page a little more fleshed out. This should probably be in the CONTRIBUTING.md but all changes that go into master should be OK to go into production.

Can you rally a few people to get some resources for this page?

I think initially we can just have a few sections with 1 or 2 links under them.

I'd like:
* a brief summary of the language. Something like what you can do with android development and a really high level overview.
* a getting started section with some links on things like how to get your environment setup
* a resources section with links to beginner oriented docs. Things like hello worlds, books, blogs, videos etc.

**Comment by jdspringr07** [comment]
@rickr Sure, I can take a look into all of that. Is getting-started going to be a landing pages for all of the areas? Also, are we going to create a getting-started controller and view container to house these?

**Comment by rickr** [comment]
I'm not completely sure yet but I know it definitely makes sense to namespace these out so we have the option and things are logically grouped.

**Comment by rickr** [comment]
Hey Josh - going to close this PR since there's a bit more work to be done. Feel free to open a new one once you've addressed the stuff above.

I've assigned you to #119. She's all yours!

**Pull request closed without merging**